packages/xo-server/src/xo-mixins/vmware/index.mjs

```diff
@@ -139,7 +139,14 @@ export default class MigrateVm {
       }
       coldChainsByNodes[key] = chainCopy
     })
-
+    // ensure the session stays alive
+    const interval = setInterval(
+      async () => {
+        await esxi.getTransferableVmMetadata(vmId)
```

Comment:
The getTransferableVmMetadata() is only used as a sort of ping?
Or has it a real use to be called to prevent the timeout?

Reply:
it is only to keep the session alive, like a ping.